tackle-test-generator-unit/src/org/konveyor/tackle/testgen/model/RapidTypeAnalysis.java

```diff
@@ -113,9 +113,13 @@ private Set<String> performAnalysis(URLClassLoader urlLoader) {
 					if (method.isAbstract() || method.isNative()) {
 						continue;
 					}
-					
-					Body body = method.retrieveActiveBody();
-					
+					Body body = null;
+					try {
+						body = method.retrieveActiveBody();
+						//to figure out what is the problem
+					} catch (RuntimeException e) { // todo - Exception should be revisited
+						continue;
```

Comment:
please log a warning for this case - method body could not be retrieved

Reply:
done